apps/service_gather/lib/gather/view_state/extractions.ex

```diff
@@ -0,0 +1,8 @@
+defmodule Gather.ViewState.Extractions do
```

Comment:
Do these need to be in separate files if they're essentially the same things and _only_ differ in the name of the collection?
This is reducing the boilerplate but it feels like a really low bar to reduce it a little further at least.

Reply:
Are you saying I should put the 3 modules in the same file? It's the same amount of code whether it's in one or three files. I don't really consider that reducing boilerplate.

Reply:
"boiler plate" may not be the best word to describe it but i think it's easier to lose track of how many of these modules there are if we need to make any changes, particularly if you add more to accommodate additional collections.

Reply:
I'm not going to make this change in this PR, but I'm going to play around with it in the next few. I'm not convinced, but I get your point and want to see it in code.